packages/wdio-cli/src/launcher.js

```diff
@@ -122,15 +112,31 @@ class Launcher {
          * schedule test runs
          */
         let cid = 0
-        for (let capabilities of caps) {
+        if (this.isMultiremote) {
+            /**
+             * Multiremote mode
+             */
             this.schedule.push({
                 cid: cid++,
-                caps: capabilities,
-                specs: this.configParser.getSpecs(capabilities.specs, capabilities.exclude).map(s => ({ files: [s], retries: config.specFileRetries })),
-                availableInstances: capabilities.maxInstances || config.maxInstancesPerCapability,
-                runningInstances: 0,
-                seleniumServer: { hostname: config.hostname, port: config.port, protocol: config.protocol }
```

Comment:
This line isn't needed for multi remote?

Reply:
It was not used before, most likely it is not needed because this line might differ per Multiremote instance